jobs/JREGIONAL_MAKE_ICS

```diff
@@ -57,10 +57,14 @@ for the FV3 (in NetCDF format).
 #
 if [ $RUN_ENVIR = "nco" ]; then
     export INPUT_DATA="${COMIN}"
+    export INPUT_DATA_NWGES="${NWGES_DIR}${SLASH_ENSMEM_SUBDIR}/ics"
 else
     export INPUT_DATA="${COMIN}${SLASH_ENSMEM_SUBDIR}/INPUT"
+    export INPUT_DATA_NWGES="${NWGES_DIR}${SLASH_ENSMEM_SUBDIR}/ics"
 fi
 mkdir_vrfy -p "${INPUT_DATA}"
+mkdir_vrfy -p "${INPUT_DATA_NWGES}"
```

Comment:
The NCO implementation uses NWGES directories to store ics/lbcs. To save disk space, I decided to store only symlinks to actual lbcs files in this case.

Reply:
Moving forward, we will need to be careful with NCO implementations and the use of symlinks.  With the move to WCOSS2, NCO is no longer wanting symlinks to be used.  Notes from Steven Earle in global-workflow:

```
The issue described in the ticket goes much further than just the recent wave issue; I can update the bugzilla if that's desired.
DATA needs to be self contained for several reasons....
-- Portability
-- Contained IO, making management of the system/storage possible
-- Pristine place to save after failures for debug/troubleshooting later
```
While storing symlinks in develop is fine, storing symlinks for the official release will likely not work.